scripts/docker/zeppelin-quick-start/README.md

```diff
@@ -27,3 +27,81 @@ docker compose -f docker-compose-zeppelin-only.yml stop
       ZEPPELIN_JMX_PORT: 9996 # Port number which JMX uses
       ZEPPELIN_MEM: -Xmx1024m -XX:MaxMetaspaceSize=512m # JVM mem options
 ```
+
+### Zeppelin With Apache Spark
+#### Install Spark Binary File
+```bash
+cd scripts/docker/zeppelin-quick-start
+wget https://archive.apache.org/dist/spark/spark-3.5.2/spark-3.5.2-bin-hadoop3.tgz
+tar -xvf spark-3.5.2-bin-hadoop3.tgz
```

Comment:
why do we need to install spark out side of the container?

Reply:
Oh, I see, you mount the Spark binary into the container later

Reply:
@pan3793 
I used the mounting method because I wanted to recommend running it according to the Spark version.

Reply:
@pan3793 ping :)